spec.md

```diff
@@ -51,13 +51,14 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 
 The high-level components of the spec include:
 
-* [Image Manifest](manifest.md) - Describes the components that make up a container image.
-* [Image Index](image-index.md) - An index of annotated image manifests distributed as a complete unit.
-* [Image Layout](image-layout.md) - The filesystem layout representing the contents of an image
+* [Image Manifest](manifest.md) - a document describing the components that make up a container image
+* [Image Index](image-index.md) - an index of annotated image manifests distributed as a complete unit
+* [Image Layout](image-layout.md) - a filesystem layout representing the contents of an image
 * [Filesystem Layers](layer.md) - changesets that describe a container's filesystem
-* [Image Configuration](config.md) - The document determining layer ordering and configuration of the image suitable for consumption by runtime-spec
-* [Descriptors](descriptor.md) - Describes the type, metadata and content address of referenced content.
+* [Image Configuration](config.md) - a document determining layer ordering and configuration of the image suitable for consumption by [runtime-spec][]
```

Comment:
runtime-spec isn't really the consumer (that would probably be the runtime implementation).  How about:

> … suitable for translation into a [runtime bundle][…].